packages/itwin/grouping-mapping-widget/README.md

```diff
@@ -29,3 +29,45 @@ import { GroupingMappingProvider } from "@itwin/grouping-mapping-widget";
   uiProviders={[new GroupingMappingProvider()]}
 />
 ```
+
+## UI Configuration
+
+We provide interface to define your own group creation method. You can use your own UI component to create/update query and visualize in the viewer accordingly.
+
+Here is an example of group custom UI provider:

```

Comment:
Instead of writing this example in README file, you could simply add a link to the `ManualUIProvider.tsx` file, which actually does something. I don't even know if this example would be a valid UI component.